src/Native/System.Private.TypeLoader.Native/amd64/ConstrainedCallSupportHelpers.S

```diff
@@ -0,0 +1,35 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+.intel_syntax noprefix
+
+#if defined(__APPLE__)
```

Comment:
Why not just include the header file with these?

Reply:
Duplicated because the Windows assembly files duplicate it too. The headers technically "belong" to the runtime and shouldn't just be included here. Whoever wrote the Windows files didn't think it's worth their time refactoring things.

Reply:
I guess that the Windows files have it duplicated because of it was pain to include a shared definition in razzle. I think `#include <../../Runtime/unix/unixasmmacros.inc>` for the dummy Unix files would be less worse than duplicating it.

Reply:
They felt equally bad to me (referring to internal implementation detail of the runtime from a file that belongs to the class library), but I changed it per your preference.